DS-173: Add base content files for all components (#308)
gsoft-inc/wl-hopper: apps/docs/content/components/concepts/Styling.mdx

    fontSize="1.375rem"
    fontWeight={680}
    paddingTop={160}
    paddingBottom={160}
    marginBottom={160}
    color="samoyed"
    borderRadius="rounded-md"
    backgroundColor="primary"
>
    Style properties are fun.
</Div>
```

### Shorthands

Props like border and paddingX are also provided to help you save keystrokes. An exhaustive list of all the supported props is available in the ["Properties List"](#Properties-List) section.

{/* TODO: example */}
```tsx
<Div paddingY={80} border="primary">
    Shorthands.
</Div>
```

### Escape hatches

 TODO
using the `UNSAFE_` properties

### Pseudo-classes

Style props doesn't support every CSS pseudo-classes. [Location pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes#location_pseudo-classes), [input pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes#the_input_pseudo-classes), [tree-structural pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes#tree-structural_pseudo-classes), ::before and ::after are not supported and will most likely never be.

When a CSS pseudo-class is not supported by Hopper style props, we recommend using a CSS class.

> Since the following user action pseudo-classes are often used, some style props supports them. These behaves like their pseudo CSS counterparts.

Reviewer: ```suggestion
Since the following user action pseudo-classes are often used, some style props support them. These behaves like their pseudo CSS counterparts.
```
I'm not exactly sure what you're trying to say here though. Maybe it could be re-worded?
What are the style props supporting exactly?

{/* TODO: table */}
|Suffix | Pseudo state |
|---|---|
| active | :active |
| hover | :hover |
| focus | :focus |

> Not all style props supports user action pseudo-classes. Find out which props support user action pseudo-classes in the ["Properties List"](#Properties-List) section.

Reviewer: ```suggestion
Not all style props support user action pseudo-classes. Find out which props support user action pseudo-classes in the ["Properties List"](#Properties-List) section.
```

## HTML Elements

> You might wonder how will you use Hopper style props on your HTML elements?

Reviewer: ```suggestion
You might wonder how to use Hopper style props on your HTML elements.
```

> Hopper provides a set of HTML elements components already configured with Hopper styled system. You should chose these components over native HTML elements.

Reviewer: ```suggestion
Hopper provides a set of HTML element components already configured with Hopper styled system. You should choose these components over native HTML elements.
```
